packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -122,6 +122,79 @@ import 'basic_types.dart';
 /// )
 /// ```
 ///
+/// ### Custom Fonts
+///
+/// Custom fonts can be declared in the `pubspec.yaml` file as shown below:
+///
+///```yaml
+/// flutter:
+///   fonts:
+///     - family: Raleway
+///       fonts:
+///         - asset: assets/fonts/Raleway-Regular.ttf
```

Comment:
Maybe remove the `assets` part of the path to be consistent with the samples below?